src/test/java/com/rockwell/basics/animals/LionTest.java

```diff
@@ -0,0 +1,41 @@
+package com.rockwell.basics.animals;
+
+import com.rockwell.basics.animals.mammal.Lion;
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+
+public class LionTest {
+
+    Lion lion = new Lion();
+
+    @Test
+    public void shouldWeightAsMuchAsEat() {
+        //given
+        lion.setWeight(10.0);
+        //when
+        lion.eat(10);
+        //then
+        assertEquals("Error when eating", 10.0, lion.getWeight());
+    }
+
+    @Test
+    public void shouldWeightLessThanEat() {
+        //given
+        lion.setWeight(5.0);
+        //when
+        lion.eat(10);
+        //then
+        assertEquals("Error when eating", 5.0, lion.getWeight());
+    }
+
+    @Test
+    public void shouldWeightMoreThanEat() {
+        //given
+        lion.setWeight(4.0);
```

Comment:
Is it correct to store test data inside test or within some kind of standalone properties files? Short answer: that depends:) Check this link https://livebook.manning.com/book/junit-recipes/chapter-5/9